.circleci/config.yml

```diff
@@ -176,10 +176,8 @@ jobs:
             npm link @audius/service-commands
             cd /home/circleci/project/service-commands/scripts/
             sudo "$(which node)" hosts.js add
-            sudo curl -L "https://github.com/docker/compose/releases/download/1.24.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+            sudo curl -L "https://github.com/docker/compose/releases/download/1.29.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```

Comment:
Turns out we were using the wrong .env file before because > docker-compose 1.28, will look at the folder the compose is invoked from for the env file (--env-file) rather than the folder the compose is in. 
https://docs.docker.com/compose/environment-variables/#the-env-file